test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Assume:
 - `BlogSerializer` has `default_include 'posts'`
 - `PostSerializer` has `default_includes 'author'`
 - Blog has_many Posts, Post has many authors, and belongs_to a category

One.) `render json: Blog.first`
```
data: 
  blog
included:
  posts
```

Two.) `render json: Blog.first, include: 'posts'`
```
data: 
  blog
included:
  posts
```

Three.) `render json: Blog.first, include: 'posts.category'`
```
data: 
  blog
included:
  posts
  categories
```